Dockerfile

```diff
@@ -1,24 +1,17 @@
-FROM node:8.12.0-slim
-
-WORKDIR /usr/src/app
-
-COPY package.json yarn.lock ./
-RUN curl -o- -L https://yarnpkg.com/install.sh | bash -s \
-    && export PATH="$HOME/.yarn/bin:$HOME/.config/yarn/global/node_modules/.bin:$PATH" \
+# ---- Base Image ----
+FROM hmcts.azurecr.io/hmcts/base/node/stretch-slim-lts-8 as base
+RUN export PATH="$HOME/.yarn/bin:$HOME/.config/yarn/global/node_modules/.bin:$PATH" \
     && yarn install --production \
     && yarn cache clean
+COPY package.json yarn.lock ./
```

Comment:
Not sure I understand this bit.  Surely we need the package.json and yarn.lock before we do the install?  And we've lost the yarn upgrade.

Reply:
I agree, just a line swap should fix it